src/dc_imex.rs

```diff
@@ -459,15 +459,13 @@ pub unsafe fn dc_decrypt_setup_file(
 
     let mut payload: *mut libc::c_char = 0 as *mut libc::c_char;
     fc_buf = dc_strdup(filecontent);
-    if !(0
-        == dc_split_armored_data(
-            fc_buf,
-            &mut fc_headerline,
-            0 as *mut *const libc::c_char,
-            0 as *mut *const libc::c_char,
-            &mut fc_base64,
-        )
-        || fc_headerline.is_null()
+    if !(!dc_split_armored_data(
```

Comment:
remove "!!"

Reply:
Fixed, but it is not as simple as just removing two `!`